dir.targets

```diff
@@ -41,12 +41,18 @@
   <Target Name="Test" />
 
   <Target Name="EnsureDependenciesDeployed"
-          BeforeTargets="Build">
+          AfterTargets="_RestoreBuildToolsWrapper">
     <MSBuild Projects="$(RepoRoot)\targets\DeployDependencies.proj"
              Targets="DeployDependencies"
-             BuildInParallel="true" />
+             BuildInParallel="true"
+             Condition="'$(OsEnvironment)'=='Windows_NT'"
+    />
+    <MSBuild Projects="$(RepoRoot)\targets\LinuxDeployDependencies.proj"
```

Comment:
The LinuxDeployDependencies.proj is a temp override of some targets to allow using the PrereleaseResolveNuGetPackageAssets task to generate the list of runtime files and to use dnu instead of nuget. At some point it will go away when we unify on a single version of the Assets task and when NuGet.exe works as expected. I though it's cleaner to create an override instead of changing DeployDependencies.proj (the override file can be deleted later), but you could go either way.